assert/assertions.go

```diff
@@ -553,11 +553,12 @@ func Len(t TestingT, object interface{}, length int, msgAndArgs ...interface{})
 	}
 	ok, l := getLen(object)
 	if !ok {
+		//TODO: should it be - \"%s\" could not be applied <to> builtin len() or maybe "len() could not be applied to \"%s\" 
 		return Fail(t, fmt.Sprintf("\"%s\" could not be applied builtin len()", object), msgAndArgs...)
```

Comment:
Isn't it - the application of the function ƒ to its argument x? len() applied to object? There could be something I'm missing...

Reply:
Yes, I think it should say: "Could not apply len() to \"%s\"".